From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Sun, 9 May 2021 22:39:21 +0900
Subject: Accept Debian build flags

Forwarded: not-needed
---
 CPP/7zip/7zip_gcc.mak | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CPP/7zip/7zip_gcc.mak b/CPP/7zip/7zip_gcc.mak
index 46732ee..0861808 100644
--- a/CPP/7zip/7zip_gcc.mak
+++ b/CPP/7zip/7zip_gcc.mak
@@ -70,7 +70,7 @@ endif
 
 else
 
-LDFLAGS = $(LDFLAGS_STATIC)
+LDFLAGS = $(DEB_LDFLAGS) $(LDFLAGS_STATIC)
 # -s is not required for clang, do we need it for GGC ???
 # -s
 
@@ -118,7 +118,7 @@ endif
 
 
 
-CFLAGS = $(MY_ARCH_2) $(LOCAL_FLAGS) $(CFLAGS_BASE2) $(CFLAGS_BASE) $(CC_SHARED) -o $@
+CFLAGS = $(DEB_CFLAGS) $(DEB_CPPFLAGS) $(MY_ARCH_2) $(LOCAL_FLAGS) $(CFLAGS_BASE2) $(CFLAGS_BASE) $(CC_SHARED) -o $@
 
 
 ifdef IS_MINGW
@@ -146,7 +146,7 @@ CXX_WARN_FLAGS =
 #-Wno-invalid-offsetof
 #-Wno-reorder
 
-CXXFLAGS = $(MY_ARCH_2) $(LOCAL_FLAGS) $(CXXFLAGS_BASE2) $(CFLAGS_BASE) $(CXXFLAGS_EXTRA) $(CC_SHARED) -o $@ $(CXX_WARN_FLAGS)
+CXXFLAGS = $(DEB_CXXFLAGS) $(DEB_CPPFLAGS) $(MY_ARCH_2) $(LOCAL_FLAGS) $(CXXFLAGS_BASE2) $(CFLAGS_BASE) $(CXXFLAGS_EXTRA) $(CC_SHARED) -o $@ $(CXX_WARN_FLAGS)
 
 all: $(O) $(PROGPATH)
 
@@ -154,7 +154,7 @@ $(O):
 	$(MY_MKDIR) $(O)
 
 $(PROGPATH): $(OBJS)
-	$(CXX) -o $(PROGPATH) -s $(MY_ARCH_2) $(LDFLAGS) $(LD_arch) $(OBJS) $(MY_LIBS) $(LIB2)
+	$(CXX) -o $(PROGPATH) $(MY_ARCH_2) $(LDFLAGS) $(LD_arch) $(OBJS) $(MY_LIBS) $(LIB2)
 
 #	-s strips debug sections from executable in GCC
 
