From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Tue, 14 Sep 2021 23:51:04 +0900
Subject: Disable NEON ABI of armel architecture

---
 C/AesOpt.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/C/AesOpt.c b/C/AesOpt.c
index 1bdc9a8..dee8a39 100644
--- a/C/AesOpt.c
+++ b/C/AesOpt.c
@@ -545,7 +545,9 @@ VAES_COMPAT_STUB (AesCtr_Code_HW)
 #if defined(_MSC_VER) && defined(MY_CPU_ARM64)
 #include <arm64_neon.h>
 #else
+# ifndef DEBIAN_NO_NEON
 #include <arm_neon.h>
+# endif
 #endif
 
 typedef uint8x16_t v128;
